core/src/com/unciv/UncivGame.kt

```diff
@@ -23,7 +23,8 @@ class UncivGame(
         private val crashReportSender: CrashReportSender? = null,
         val exitEvent: (()->Unit)? = null,
         val cancelDiscordEvent: (()->Unit)? = null,
-        val fontImplementation: NativeFontImplementation? = null
+        val fontImplementation: NativeFontImplementation? = null,
+        val consoleMode: Boolean = false
```

Comment:
There are too many parameters here.
Please turn this into a class, like UncivGameParameters or something.

Reply:
Ok, undesrtood.